tests/DeserializerFactory.tests.js

```diff
@@ -21,14 +21,14 @@ QUnit.test( 'registerDeserializer(), newDeserializerFor()', function( assert ) {
 		}
 	];
 
-	assert.throws(
+	assert['throws'](
```

Comment:
As I was trying to say on the other PR. I'd rather ignore dot-notation rule in tests and keep the "old" form here. assert['throws'] looks silly given assert.ok etc are "fine" by eslint.